sympy/stats/joint_rv_types.py

```diff
@@ -394,6 +394,105 @@ def MultivariateEwens(syms, n, theta):
     """
     return multivariate_rv(MultivariateEwensDistribution, syms, n, theta)
 
+#-------------------------------------------------------------------------------
+# Generalized Multivariate Log Gamma distribution ---------------------------------------------------------
+
+class GeneralizedMultivariateLogGammaDistribution(JointDistribution):
+
+    _argnames = ['omega', 'v', 'l', 'mu']
```

Comment:
Shouldn't these be `delta`, `v`, `l` and `mu`?
Also, I'd expect G-MVLG to return a random vector. Note that `omega` can not be any arbitrary matrix as it is defined by the correlation among the random variables constituting the random vector.

Reply:
Actually delta is created by omega itself, so I thought of accepting omega rather than delta to ease the work of the end user.
Checks on entries of omega have been added for correct correlation coefficient. 
G-MVLG returns an indexed joint random variable so it's a vector as far as I can figure out.

Reply:
CC @Upabjojr

Reply:
What about allowing both parameters to be specified? Omega does not appear in the PDF. Maybe the class should handle `delta` only, while the function constructor should accept both possibilities.